### PR TITLE
Change default GET / route to return a 200

### PR DESCRIPTION
**Pull request by phamann**
### TL;DR
The current version of the template will return a `500` response when you hit the default `GET /` route, this is also the most common request a user will make first during on-boarding. Therefore, it doesn't lead to a nice first experience with the platform. After listening to this feedback in various user sessions it is better if provide a nice welcome message instead, this will also help reassure the user that they have successfully deployed their first C@E application. 

### What
- Add new synthetic 200 response with friendly welcome/success message. 
- Fix comment punctuation.
- Run `cargo fmt`.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -29,8 +25,13 @@ fn handle_request(mut req: Request<Body>) -> Result<Response<Body>, Error> {
 
     // Pattern match on the request method and path.
     match (req.method(), req.uri().path()) {
-        // If request is a `GET` to the `/` path, send to a named backend
-        (&Method::GET, "/") => {
+        // If request is a `GET` to the `/` path, send a default response.
+        (&Method::GET, "/") => Ok(Response::builder()
+            .status(StatusCode::OK)
+            .body(Body::try_from("Welcome to Fastly Compute@Edge!")?)?),
```
Comment (phamann) [inline src/main.rs:31]: What should this say? Is this ok? Other options:
- Success!
- It works!
- You have successfully deployed the starter kit 
- etc

Thoughts / suggestions welcome

Comment (denen99) [reply src/main.rs:31]: I think that's fine. Other suggestions, 

"Congratulations!  Welcome to Compute@Edge" 
"Hello, from your new WASM program at Compute@Edge!"
"Success.  Your new WASM programing is running on Compute@Edge!"

Comment (triblondon) [reply src/main.rs:31]: I think the original proposal is great 👍

**Review by triblondon — APPROVED** [review]

File: src/main.rs (hunk 1 above)
Comment (phamann) [reply src/main.rs:31]: Ok am going to stick with original. Thanks for the other suggestions.
